libs/ledger-live-common/src/families/cosmos/js-getTransactionStatus.ts

```diff
@@ -66,7 +51,9 @@ export class CosmosTransactionStatusManager {
       t.validators.some(
         (v) =>
           !v.address ||
-          !v.address.includes(this._validatorOperatorAddressPrefix)
+          !v.address.includes(
+            cryptoFactory(a.currency.id).validatorOperatorAddressPrefix
```

Comment:
you should create a variable `validatorOperatorAddressPrefix` at the top of the function and reuse it here instead of calling `cryptoFactory` twice in the function (line 55 and 152)

Reply:
yes, you are right.
It is a minor issue. the "cryptoFactory" is very light. I'll do a small refactoring at the end. Thanks

Reply:
oh, no, it is impossible. Because they are in two different functions, we can't merge these two variables.